README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@antoviaque Responding to your request from [OC-2608](https://tasks.opencraft.com/browse/OC-2608):

> There are often slightly different things that we need to do with each mass re-deployment, and I think its useful to be able to customize the behavior each time.

> Alright, can we list these things precisely, so that i can become a todo list for the next iterations?

@bdero's [comment](https://github.com/open-craft/opencraft/pull/207#discussion_r115268569) covers the some of the immediate steps that we need to take in order to turn the script into a management command:

> For making this reference into a management command, step 4 and 5 would need to be reworked so that they will block until the low priority huey queue is empty; I'm not sure how to do this but I'm sure it can be done.

... except that I think it should be step 3 and 4 that should be blocking; step 5 does not involve any asynchronous processing.

Beyond that, we should parameterize the management command on:

* The main tag name to use (`ficus3-redeployment` in the script). We can derive success and failure tags from that name automatically.
* The source release to use for selecting instances (`open-release/ficus.1` in the script).
* The target release to upgrade to (`open-release/ficus.3` in the script).

Lastly, to avoid the possibility of being left with some instances that failed to upgrade after the command finishes, we could modify step 4 to iterate until the set of instances currently tagged with `failure_tag` is empty.

CC @bradenmacdonald @bdero @smarnach